build/platform_and_feature_flags.props

```diff
@@ -26,4 +26,9 @@
   <PropertyGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkNetStandard)'">
     <DefineConstants>$(DefineConstants);NETSTANDARD;SUPPORTS_CONFIDENTIAL_CLIENT;SUPPORTS_BROKER;SUPPORTS_CUSTOM_CACHE;SUPPORTS_OS_SYSTEM_BROWSER;SUPPORTS_WEBVIEW2</DefineConstants>
   </PropertyGroup>
+
+  <PropertyGroup Condition="$([MSBuild]::IsOsPlatform('Windows'))">
```

Comment:
This is a build time check. We always build on Windows, so the condition is always true and pretty much useless :), i.e. the flag will be available even on Xamarin etc. 

![image](https://user-images.githubusercontent.com/12273384/117310221-fa173480-ae7a-11eb-96c4-d5be38e53ab7.png)

Reply:
To enable this Kerberos on desktop OS supporting Windows, we need to add this flag to .NET Fx, .NET Core, .NET 5, .NET Standard and UWP. @yoko-ms  - do you expect this to work on UWP ?

Secondly, for .NET Core and .NETStandard targets, we need to further check the OS using a helper like this [one](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/1770654e7a2985dc06849366e5ae8e229b22faa7/src/client/Microsoft.Identity.Client/Platforms/Features/DesktopOs/DesktopOsHelper.cs#L23)

@yoko-ms - I can help make this changes.

Reply:
Thanks for pointing. The AAD Kerberos feature can be supported in all platform/OS excepts the Ticket Cache support feature which has dependency on Windows. Based on this, I figured out that I can use (iOS || MAC || ANDROID) combination as a replacement of SUPPORT_KERBEROS.
- Removed and replaced to use existing (iOS || MAC || ANDROID) definition.